iam/principal_service.go

```diff
@@ -124,7 +124,7 @@ func tokenToPrincipal(untypedPrincipal interface{}, scopes interface{}) (*Armory
 	if scopes != nil {
 		scopeStr, ok := scopes.(string)
 		if ok {
-			typedPrincipal.Scopes = strings.Split(scopeStr, " ")
+			typedPrincipal.Scopes = append(typedPrincipal.Scopes, strings.Split(scopeStr, " ")...)
```

Comment:
Is there a reason we want to absorb the Auth0 scopes in?

Reply:
Yes - when we do RBAC for client credentials, we will want `scopes` to include both the old Auth0 scopes (`read:customerConfiguration`) and our new RBAC scopes.